### ISSUE TITLE
Add setting to choose whether consumable items have Auto Use and Auto Destroy checked or unchecked by default

### ISSUE THREAD

**Issue by cswendrowski** [foundryvtt/pf2e#1451]
In GitLab by @danielagardeazabal on May 30, 2021, 02:10

Currently, Auto Use and Auto Destroy are always checked by default in a consumable's details page. Personally, I never want either of these options to be checked, so it is a major hassle to have to uncheck them every single time someone gets a consumable. Ideally, there should be a setting that lets the GM choose the default option to be either checked or unchecked. If that is unfeasible for compendium reasons, then I at least propose changing the default setting to be that Auto Use and Auto Destroy are unchecked.

Reasons: Players always want to just share the descriptions of their consumables with the rest of the party, without actually committing to using the item, but they can't do that if posting a consumable item to chat also removes it from their inventory. Furthermore, in many cases the party decides not to use that item anyway, but if the item was automatically removed, the onus shouldn't be on the player or GM to remember to re-add the item to their inventory. It is difficult to remember to re-add the item, since in a person's mindset, they shouldn't need to re-add anything. It just shouldn't have disappeared in the first place. Unless the GM perfectly remembers to manually uncheck these options each time a consumable is added, the current system leads to many items "disappearing" from a characters inventory.

On the other hand, if Auto Use and Auto Destroy are unchecked by default, it is much easier for a player to remember to manually remove an item from their inventory, since they know they just used it. In addition, even if a player forgets to manually remove the item, a player having an extra consumable in their inventory "feels" more fair than a player missing an item they are supposed to have. Lastly, even if a player forgets to manually remove an item, it is likely that they will later realize they should remove it, after they look at their inventory again in the future.

In conclusion, I hypothesize that the number of inventory errors is much higher if Auto Destroy and Auto Use are checked than if they are unchecked. Thus, the current system is the worst of the 3 possible options (checked by default < unchecked by default < a system setting lets the GM choose whether they are checked or unchecked by default).
